### ISSUE TITLE
Tensor Classification

### ISSUE THREAD

**Issue by Haobo1108** [tensorly/tensorly#155]
#### Is your feature request related to a problem? Please describe.
<-- I'm always frustrated when I wonder to predict the type of an object. Usually, the information of the object is high dimensional or can be summarized in a tensor plus some covariates. -->

#### Describe the solution you'd like
<-- The classification problem can be transformed into multinomial logistic regression of the softmax regression problem. -->

#### Additional context
<-- If this feature is easy to achieve by changing some parts of the current codes. Could you please provide some guidance. I think the main part is how to change the loss function and whether the current algorithm is suitable.-->

**Comment by JeanKossaifi** [comment]
I am not sure I understand what you mean -- could you explain?
You want to do classification instead of regression?

**Comment by Haobo1108** [comment]
Dear Jean,

Yes. I want to do classification rather than regression. However, there are many ways to do classification and I think it is not easy to include all of them in a package. Therefore, I closed the issue.

Currently, I am using your package to do tensor classification by using MNL. Tensorly is really helpful. Thank you!

**Comment by JeanKossaifi** [comment]
We are definitely interested in having at least the most important tensor classification methods in TensorLy, if you are interested in creating a Pull-Request! :)
